jaguar_serializer_cli/test/serializer/fromMap/nested_list_test.dart

```diff
@@ -27,8 +27,14 @@ void main() {
           expect(
```

Comment:
Don't need `_test` suffix to files. All files under those directory are tests.

Reply:
yes we need, we don't need to maintain root file and that way we get from wich file the error comes, this a good practice from dart styleguide